plugins/module_utils/foreman_helper.py

```diff
@@ -657,7 +657,8 @@ def list_resource(self, resource, search=None, params=None):
 
         if search is not None:
             params['search'] = search
-        params['per_page'] = PER_PAGE
+        if not resource == 'snapshots':
+            params['per_page'] = PER_PAGE
```

Comment:
Why is this needed? The API reply below in the fixtures does contain the `page` and `per_page` attributes, so I would expect the API to support paging?

Reply:
:thinking: that's weird!

Would you mind dropping that change, re-record, and push that as a separate commit so that I can look at the diff?

Reply:
Hah!

I think I know what's up. Could it be that the apidoc snapshot (lol) we're using is older than https://github.com/ATIX-AG/foreman_snapshot_management/commit/7c91a1e9e7db3c34249d37543246e1ebc4acae2d?

```
% git log tests/fixtures/apidoc/snapshot.json
commit 241d49fd27e7f9926373a442768f12cbc2273e4e
Author: Manisha Singhal <manisha1595@gmail.com>
Date:   Thu Jun 27 14:57:05 2019 +0200

    Add Snapshot module
```

but the commit above is Dec 2019.

Just update the apidoc file and we're good! :)